frontend/e2e/fixtures/import-file/partial-metrics-show-chart.ts

```diff
@@ -0,0 +1,100 @@
+export const partialMetricsShowChart = {
+  projectName: 'Heartbeat Metrics',
```

Comment:
update project name to the test case name, will be more efficient

Reply:
done